gptresearcher/actions/web_search.py

```diff
@@ -7,36 +7,34 @@
 from langchain.utilities import SearxSearchWrapper
 from config import Config
 
-CFG = Config()
 
-
-def web_search(query: str, num_results: int = 4) -> str:
+def web_search(researcher, query: str, num_results: int = 4) -> str:
```

Comment:
@gregdrizz same feedback here. don't pass a researcher object, add the relevant params to function. If there are many consider user **kwargs to make the function cleaner. Please keep all util functions generic to gptresearcher object